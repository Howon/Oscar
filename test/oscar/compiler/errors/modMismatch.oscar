def main() => unit = {
  Println(3.0 % 2.0);
}