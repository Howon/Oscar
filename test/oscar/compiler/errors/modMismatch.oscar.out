operand type mismatch: double double